### PR TITLE
fix: [CDS-86780]: Change text Setup to set up as per Docs team

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by namanharness — APPROVED** [review]

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@ homepage:
   slogan: '{{ $.common.purpose.cf.descriptionOnly }}'
   learnMore: Learn more about Feature Flags
 selectFlagRepo:
-  introText: 'Git Experience has already been enabled for your project. Please confirm your Git setup before creating feature flags.'
+  introText: 'Git Experience has already been enabled for your project. Please confirm your Git set up before creating feature flags.'
```
Comment (michaelcretzman) [inline src/modules/75-cf/strings/strings.en.yaml:96]:
```suggestion
  introText: 'Git Experience has already been enabled for your project. Please confirm your Git setup before creating feature flags.'
```

File: src/modules/75-cd/strings/strings.en.yaml [hunk 2]
```diff
@@ -1389,4 +1389,4 @@ asgPlaceholder: Enter ASG Name
 asgLoadBalancer: ASG Load Balancer
 scopeToSpecificServices: Scope to Specific Services
 selectedPrimaryArtifact: 'Selected Primary Artifact: {{artifactId}}'
-reConfigurePipelineClusters: 'Your cluster setup is out of date, please reselect the cluster(s) you want to deploy to in this PR Pipeline to continue'
+reConfigurePipelineClusters: 'Your cluster set up is out of date, please reselect the cluster(s) you want to deploy to in this PR Pipeline to continue'
```
Comment (michaelcretzman) [inline src/modules/75-cd/strings/strings.en.yaml:1392]:
```suggestion
reConfigurePipelineClusters: 'Your cluster setup is out of date, please reselect the cluster(s) you want to deploy to in this PR Pipeline to continue'
```

File: src/modules/75-cd/strings/strings.en.yaml [hunk 3]
```diff
@@ -966,7 +966,7 @@ getStartedWithCD:
           retryVerify: Retry Verification
           pipelinesuccess: The pipeline was successfully created.
     reviewAndRunStep:
-      title: Verify the setup
+      title: Verify the set up
```
Comment (michaelcretzman) [inline src/modules/75-cd/strings/strings.en.yaml:969]:
```suggestion
      title: Verify the setup
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 4]
```diff
@@ -1116,7 +1116,7 @@ onboarding:
 
       # Carthage
 
-      Carthage is intended to be the simplest way to add frameworks to your Cocoa application. Carthage builds your dependencies and provides you with binary frameworks, but you retain full control over your project structure and setup. Carthage does not automatically modify your project files or your build settings.
+      Carthage is intended to be the simplest way to add frameworks to your Cocoa application. Carthage builds your dependencies and provides you with binary frameworks, but you retain full control over your project structure and set up. Carthage does not automatically modify your project files or your build settings.
```
Comment (michaelcretzman) [inline src/modules/75-cf/strings/strings.en.yaml:1119]:
```suggestion
      Carthage is intended to be the simplest way to add frameworks to your Cocoa application. Carthage builds your dependencies and provides you with binary frameworks, but you retain full control over your project structure and setup. Carthage does not automatically modify your project files or your build settings.
```

File: src/modules/75-ci/strings/strings.en.yaml [hunk 5]
```diff
@@ -73,7 +73,7 @@ runTestsStep:
   validation:
     namespacesRequiredWhenCsharp: 'Namespaces is required in case Language is C#'
 runTestsErrorTrackingSetupText: Do you want to enable Error Tracking?
-runTestsMavenSetupTitle: Maven setup
+runTestsMavenSetupTitle: Maven set up
```
Comment (michaelcretzman) [inline src/modules/75-ci/strings/strings.en.yaml:76]:
```suggestion
runTestsMavenSetupTitle: Maven setup
```

File: src/modules/75-ci/strings/strings.en.yaml [hunk 6]
```diff
@@ -208,7 +208,7 @@ getStartedWithCI:
   nonGitCloneOption: Use this option if your Git provider is not supported by Harness or if you have nothing to clone.
   cloneGitRepo: Clone Git Repository
   cloneGitRepoHelpText: If clone is enabled, Harness will clone your Git Repository automatically when the execution starts.
-  createPipelineWithOtherOption: Please click on "Create Pipeline" below to complete the pipeline setup.
+  createPipelineWithOtherOption: Please click on "Create Pipeline" below to complete the pipeline set up.
```
Comment (michaelcretzman) [inline src/modules/75-ci/strings/strings.en.yaml:211]:
```suggestion
  createPipelineWithOtherOption: Please click on "Create Pipeline" below to complete the pipeline setup.
```

**Review by michaelcretzman — APPROVED** [review]
approved with edits

**Pull request merged**